yarn.lock

```diff
@@ -2057,15 +2057,6 @@
     call-me-maybe "^1.0.1"
     glob-to-regexp "^0.3.0"
 
-"@near-wallet/feature-flags@^0.0.4":
```

Comment:
This lock update doesn't seem quite right 🤔  

Weird that it's trying to remove `feature-flags`, maybe `yarn` get's confused when updating `packages/frontend/package.json` since `feature-flags` actually exists in the root `package.json`?

I think we can just revert this file change, I can see that `bip38-light@1.0.7` already exists and we don't want to remove these packages.

Reply:
Good catch! Reverted